radixdlt/src/main/java/com/radixdlt/properties/PersistedProperties.java

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Moved from org.radix now that it has a reasonable set of unit tests and type warnings have been removed.